app/jobs/geocode_adviser_job.rb

```diff
@@ -0,0 +1,15 @@
+require 'geocoder'
+
+class GeocodeAdviserJob < ActiveJob::Base
+  def perform(adviser)
+    point = Geocoder.coordinates(adviser.full_street_address)
+    point ? stat(:success) : stat(:failed)
+    adviser.geocode!(point)
+  end
+
+  private
+
+  def stat(key)
+    Stats.increment("radsignup.geocode_adviser.#{key}")
```

Comment:
As discussed, it would be better for us to use dots rather than underscores to denote the segments. This way we can wildcard certain sections when building graphs.